reposcan/repodata/repository_controller.py

```diff
@@ -224,6 +225,7 @@ def import_repositories(self):
                 failures += 1
         if failures > 0:
             self.logger.warning("Failed to import %d repositories.", failures)
+            FAILED_IMPORT_REPO.inc(failures)
```

Comment:
increase counter also in `def store(self):` method, this handles only repo import but not repo sync